[13.0] Facturae: Missing crons

etobella opened the pull request:
Fixes #2191 

En la migración de FACe se perdió el cron. De paso añado los scripts de migración, que cambia el modelo y al ser noupdate no se debería actualizar automáticamente.

Pasa lo mismo (lo del script) en e.Fact.

etobella committed:
[FIX] l10n_es_facturae_face: Add lost job

Also adds a migration script for a change of model
Fixes #2191

etobella committed:
[OU] l10n_es_facturae_efact: Missing migration script

The cron job needs to change the model

HaraldPanten reviewed (changes requested):
¿Puedes revisar los commits?

Yo haría:

1- O bien 2 commits por separado: uno para crons y otro para scripts de migración.

2- O todo en uno, pero especificando los cambios en el mensaje del commit.

Gracias por la aportación.

etobella commented:
Los tests fallan por esto https://github.com/OCA/edi/pull/551 Esperando para hacer merge en EDI :smile:

etobella commented:
El error viene del 303...

pedrobaeza commented:
Sí, hay un error en esta rama con el OSS que no he sabido descifrar aún (no he investigado a fondo, pero tampoco tengo de primeras la posible razón).

etobella committed:
[IMP] l10n_es_facturae_face: Remove unnecessary parameter

The parameter is no longer necessary. Added a script to pass the correct data on the migration

etobella commented:
De paso, arreglo otro error que acabo de detectar con @LoisRForgeFlow (pendiente pasarlo a 14)

etobella commented:
@HaraldPanten He modificado los textos de los commits, no sé si lo ves mejor así

HaraldPanten commented:
Pues ahora sí pasan los tests...

HaraldPanten reviewed (approved):
Creo que nos hemos solapado Enric. Por mi parte OK.

etobella committed:
[OU] l10n_es_facturae_face: Fix migration script, state was not set properly

etobella committed:
[OU] l10n_es_facturae_efact: Fix migration script, state was not set properly

HaraldPanten commented:
Hola,

Creo que los dos últimos commits podrían fusionarse, ¿no?

Saludos.

etobella commented:
Son de módulos diferentes, por eso no los uní

pedrobaeza reviewed (approved):
Podemos ya darle salida creo yo

pedrobaeza commented:
/ocabot merge nobump

HaraldPanten commented:
> Son de módulos diferentes, por eso no los uní

Toda la razón.

pedrobaeza commented:
/ocabot merge nobump
